fe/fe-core/src/main/java/com/starrocks/leader/ReportHandler.java

```diff
@@ -137,6 +140,13 @@ public enum ReportType {
 
     private Map<ReportType, Map<Long, ReportTask>> pendingTaskMap = Maps.newHashMap();
 
+    /**
+     * Record the mapping of <tablet id, backend id> to the to be dropped time of tablet.
+     * We will delay the drop of tablet based on configuration `tablet_report_drop_tablet_delay_sec`
+     * if we don't find the meta of the tablet in FE.
+     */
+    private static final Table<Long, Long, Long> TABLET_TO_DROP_TIME = HashBasedTable.create();
```

Comment:
Why do you make this table as static? And it seems there is no concurrent protect for this structure.

Reply:
ReportHanlder is singleton, there is no concurrency. And because the user of this map is static method, so better use static map here.